components/dashboard_modules/SunnyTimeframe.tsx

```diff
@@ -0,0 +1,44 @@
+import { FC } from 'react';
+
+import {
+  useFutureGraphData,
+  getNextThresholdIndex,
+  graphThreshold,
+  Value,
+} from 'lib/utils';
+
+const SunnyTimeframe: FC = () => {
+  const { data } = useFutureGraphData();
+  const nextThreshold = getNextThresholdIndex(
+    data?.forecast_values,
+    graphThreshold
+  );
+
+  if (nextThreshold) {
```

Comment:
Instead of putting this all inside the `if`, maybe do an `if (!nextThreshold)` with an early `return null`.